### DISCUSSION TITLE
OneDrive folder in PC is empty

### DISCUSSION THREAD

**Discussion by Basriy** [abraunegg/onedrive#2982] [Q&A] [answered]
I run Arch both in my laptop and PC, and have installed OneDrive on both. My laptop running OneDrive without a problem, but somehow OneDrive folder in my PC doesn't show/have anything. When I run "--sync" it reports connection is established and sync is complete.

**Comment by abraunegg** [comment]
Lets start with some basic items. What is the output of on both systems:
1. `onedrive --version` ?
2. `onedrive --display-config` ?
3. `onedrive --display-sync-status` ?
4. Are you 'sharing' your home directory on both systems with a common location?

On the PC that is not syncing, what happens when you do the following:
```
onedrive --sync --verbose --resync
```

**Reply by Basriy** [reply, marked as answer]
Sorry for bothering. I just uninstalled everything related with my previous install, then went with all the installation procedures once again. I think the problem was I didn't have/properly installed D compiler (facepalm). Now everything is syncing OK. Thank you for quick response.
